tests/test_wfsim.py

```diff
@@ -66,10 +66,10 @@ def test_sim_nT():
         st = strax.Context(
             storage=tempdir,
             config=dict(
-                nchunk=1, event_rate=1, chunk_size=2,
+                nchunk=1, event_rate=10, chunk_size=5,
                 detector='XENONnT',
                 fax_config=('https://raw.githubusercontent.com/XENONnT/WFSim'
-                '/22004e28421044452f42aaf5797be7186e07bbba/files/XENONnT_wfsim_config.json'),
+                '/a412375ad1fc85f30596b2b73cd8ffe5401de42e/files/XENONnT_wfsim_config.json'),
```

Comment:
@petergaemers , I decreased this to 1 and 2 respectively to reduce the overall time we spend on CI. I figured that if it would work for a little data, it would also work for more but with less waiting time, are there many things that will only show up with running more simulations (`event_rate=10, chunk_size=5,`)?

Reply:
(stupid github makes it unclear what I was referring to but it's the:
`event_rate=1, chunk_size=2` -> `event_rate=10, chunk_size=5`

for both the nT and 1T test

Reply:
The reason is that I saw some of the S1 code not covered by the test but should be. And it's possible that LCE made S1 too small. So actually be a better solution than increase the event rate is to fix rand instruction. So we can guarantee S1 has detectable photons.

Reply:
I see, makes sense. We could also increase the energy, if we can reduce the testing time, there is clear benefit.

Reply:
I'll make the default energy in rand_instructions larger, s1s are so fast it will not matter